traits/src/identities.rs

```diff
@@ -50,6 +51,16 @@ zero_impl!(i64,   0i64);
 zero_impl!(f32, 0.0f32);
 zero_impl!(f64, 0.0f64);
 
+impl<T: Zero + PartialEq> Zero for Wrapping<T> where Wrapping<T>: Add<Output=Wrapping<T>> {
```

Comment:
Hmm.  I was worried that a generic `impl` would be a breaking change, in case someone had already implemented it for their own `Foo` and `Wrapping<Foo>`, but it looks like they couldn't do that:

```
    |
115 | impl Zero for Wrapping<Foo> {}
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ impl doesn't use types inside crate
    |
    = note: the impl does not reference any types defined in this crate
    = note: define and implement a trait or new type instead
```

... even though `Foo` is defined on the line right before.  I always forget exactly how coherence works.  I think you _can_ have this overlap with users if the custom type were a trait parameter instead.

Anyway, looks like this is OK.